runner/service-options.sls

```diff
@@ -0,0 +1,21 @@
+# This helper state can be used to deploy custom systemd options for the gitlab-runner service
+
+include:
+  - gitlab.runner
+
+# Ensure that the gitlab-runner service terminates gracefully when stopped
+# by systemd. See https://docs.gitlab.com/runner/configuration/init.html#overriding-systemd
+# The long (2h) timeout ensures that started CI jobs are allowed to run to completion.
```

Comment:
I think this should be a pillar setting, where this is also configurable and then included in `init.sls`. The filename is a bit confusing like this. 

I'd go for something like this:

```yaml
gitlab-runner:
  systemd-timeout-stop-sec: 7200
```

Then deploy this if the pillar is set. wdyt?

Reply:
implemented as suggested